packages/delegate/src/delegateToSchema.ts

```diff
@@ -190,14 +207,17 @@ function getDelegationContext({
     context,
     info,
     rootValue: rootValue ?? info?.rootValue ?? emptyObject,
-    returnType: returnType ?? info?.returnType ?? getDelegationReturnType(subschemaOrSubschemaConfig, targetOperation, targetFieldName),
+    returnType:
+      returnType ??
+      info?.returnType ??
+      getDelegationReturnType(subschemaOrSubschemaConfig, targetOperation, targetFieldName),
     transforms,
     transformedSchema: transformedSchema ?? subschemaOrSubschemaConfig,
     skipTypeMerging,
   };
 }
 
-function validateRequest(delegationContext: DelegationContext, document: DocumentNode) {
+function validateRequest(delegationContext: DelegationContext<any>, document: DocumentNode) {
```

Comment:
Wherever the actual context type wrapped in the `DelegationContext` does not matter I tried to use any, to avoid a generic parameter on the function